reg-index/src/lib.rs

```diff
@@ -72,6 +72,7 @@ pub struct IndexPackage {
     /// The name of the package.
     pub name: String,
     /// The version of the package.
+    #[serde(with = "serde_with::rust::display_fromstr")]
```

Comment:
It's not clear to me why this is adding `serde_with`. semver implements serializers/deserializers (with the serde feature).